docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplate.java

```diff
@@ -102,6 +105,7 @@
 
     @DataBoundConstructor
     public DockerTemplate(String image, String labelString,
+    		              Integer numExecutors,
```

Comment:
Please remove this from constructor and add DataBoundSetter with right default value